src/client/components/app/VaultDetail/VaultDetailPanels.tsx

```diff
@@ -108,7 +109,7 @@ const OverviewInfo = styled(Card)`
   a {
     text-decoration: underline;
     color: inherit;
-    color: ${({ theme }) => theme.colors.titles};
+    //color: ${({ theme }) => theme.colors.titles};
```

Comment:
If its no longer needed, lets remove it better :)

Reply:
yes great ty